### ISSUE TITLE
Menu - add isARIADisabled css styling for menu options

### ISSUE THREAD

**Issue by kmcfaul** [patternfly/patternfly#5528]
Currently there is no aria-disabled css for a menu option (in the case that a menu option should appear disabled but still be focusable for the purposes of a tooltip, etc). This should be similar to `Button`.
